### PR TITLE
Remove html_shim

### PR DESCRIPTION
**Pull request by jacob314**
Flutter now allows importing dart:html code as long as it is protected by a conditional import.

### CODE REVIEW

File: packages/devtools_app/lib/main.dart [hunk 1]
```diff
@@ -2,32 +2,14 @@
...
+
 import 'package:flutter/material.dart';
-import 'package:flutter/foundation.dart';
-import 'package:html_shim/html.dart' show document, window;
 
+import 'src/config_specific/flutter/framework_initialize/framework_initialize.dart';
 import 'src/flutter/app.dart';
-import 'src/framework/framework_core.dart';
 
 void main() {
```
Comment (jacob314) [inline packages/devtools_app/lib/main.dart]: the one non-mechanical change is moving this code to framework_initialize.dart and its conditional imports.

File: packages/devtools_app/lib/src/config_specific/flutter/framework_initialize/_framework_initialize_stub.dart [hunk 2]
```diff
@@ -0,0 +1,8 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Return the url the application is launched from.
+String initializePlatform() {
+  throw UnimplementedError('Unsupported platfrom');
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/config_specific/flutter/framework_initialize/_framework_initialize_stub.dart]: nit: add more details to message . this also matches the messages for other config specific files like this

"Attempting to initialize framework for unrecognized platform"

File: packages/devtools_app/lib/src/config_specific/flutter/framework_initialize/_framework_initialize_stub.dart [hunk 3]
```diff
@@ -0,0 +1,8 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/config_specific/flutter/framework_initialize/_framework_initialize_stub.dart]: nit: rename file to _fake_framework_initialize to match other config specific files.

File: packages/devtools_app/lib/src/config_specific/flutter/framework_initialize/_framework_initialize_stub.dart (hunk 2 above)
Comment (jacob314) [reply packages/devtools_app/lib/src/config_specific/flutter/framework_initialize/_framework_initialize_stub.dart]: done

File: packages/devtools_app/lib/src/config_specific/flutter/framework_initialize/_framework_initialize_stub.dart (hunk 3 above)
Comment (jacob314) [reply packages/devtools_app/lib/src/config_specific/flutter/framework_initialize/_framework_initialize_stub.dart]: I think this naming pattern is better. I'd like to update the other libraries to match this unless there is a good reason for the current convention.
With _fake last all the implementations will have a common prefix which makes it easier to catch typos in the names, etc.

**Pull request merged**